src/features/events/components/SelectionBarEllipsis.tsx

```diff
@@ -1,14 +1,16 @@
 import { useContext } from 'react';
-import { useRouter } from 'next/router';
 
-import { EventsModel } from '../models/EventsModel';
 import messageIds from '../../calendar/l10n/messageIds';
 import { resetSelection } from '../store';
+import useEventsMutations from '../hooks/useEventsMutations';
 import { useMessages } from 'core/i18n';
-import useModel from 'core/useModel';
 import { ZUIConfirmDialogContext } from 'zui/ZUIConfirmDialogProvider';
 import ZUIEllipsisMenu from 'zui/ZUIEllipsisMenu';
-import { useAppDispatch, useAppSelector } from 'core/hooks';
+import {
+  useAppDispatch,
```

Comment:
The fact that `useAppDispatch()` is "exposed" in this component makes me think that there is an opportunity for further refactoring here, in order to _encapsulate_ the logic of bulk operations more. But that's not something we need to do for this PR.